pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -465,22 +465,16 @@ func (adc *attachDetachController) populateDesiredStateOfWorld() error {
 					err)
 				continue
 			}
-			attachState := adc.actualStateOfWorld.GetAttachState(volumeName, nodeName)
-			if attachState == cache.AttachStateAttached {
```

Comment:
@jsafrane, @msau42 , @houjun41544 Does removing this condition (_where the code checks if pod.nodename is in the attached node_) solves the problem and restores the volume specs in ASW when attach/detach controller starts.